radix-engine/src/object_modules/metadata/package.rs

```diff
@@ -16,10 +16,9 @@ use super::{RemoveMetadataEvent, SetMetadataEvent};
 
 #[derive(Debug, Clone, Eq, PartialEq, ScryptoSbor)]
 pub enum MetadataError {
-    KeyStringExceedsMaxLength { max: usize, actual: usize },
-    ValueSborExceedsMaxLength { max: usize, actual: usize },
     ValueDecodeError(DecodeError),
-    MetadataValidationError(MetadataValidationError),
+    MetadataKeyValidationError(MetadataKeyValidationError),
```

Comment:
This will require temporarily using `, regenerate` to update the `VersionedLocalTransactionExecution`.

Reply:
Will do.